public/partials/buyer_profile_large.html

```diff
@@ -98,35 +100,46 @@ <h3 class="{{new_rep ? 'dark':'white'}}">Approved Employees</h3>
                             </md-button>
                         </div>
                     </div>
-                    <div ng-include="new_rep ? '/partials/representative_new.html':null">
+                    <div class="{{edit_rep ?'dark':'white'}}" ng-include="new_rep || edit_rep ? '/partials/representative_new.html':null">
 
                     </div>
+
                 </md-card-content>
-                <md-card-content>
+                <md-card-content ng-show="!edit_rep">
```

Comment:
@nick22891 not saying that this is bad... but try to use `ng-if` vs `ng-hide` or `ng-show`.  `ng-if` removes the DOM object from the html and `ng-show` and `ng-hide` relies on css to hide/show which in my experience has been unreliable.